docs/products.md

```diff
@@ -2,8 +2,55 @@
 
 ## RTC
 
+SAR datasets inherently contain geometric and radiometric distortions due to terrain
+being imaged by a side-looking instrument. Radiometric terrain correction (RTC) removes
+these distortions and creates analysis-ready data suitable for use in GIS applications.
+RTC processing is a required first step for many amplitude-based SAR applications.
+
+Sentinel-1 RTC products are generated by ASF's HyP3 processing platform leveraging
+GAMMA Software.  Products are distributed as UTM-projected GeoTIFFs with a pixel
+spacing of 30 meters. To learn more, visit the
+[ASF Sentinel-1 RTC Product Guide](guides/rtc_user.md).
+
+A Digital Elevation Model (DEM) is required for processing InSAR, and ASF uses the
+best publicly-available DEM with full coverage of the processing area,
+see [Digital Elevation Model](dems.md) for more information.
+
 ## InSAR (SDK + API only)
 
+Interferometric SAR (InSAR) uses the phase differences from repeat passes over the
+same area to identify regions where the distance between the sensor and the Earth's
+surface has changed. This allows for the detection and quantification of deformation
+or movement.
+
+Note that the imaging surface will vary depending on the sensor wavelength. Because
+Sentinel-1 is a C-band sensor, the waves will not penetrate very deeply into vegetation.
+The imagery likely represents the top of the canopy in densely vegetated areas rather
+than the actual terrain. In addition, vegetated areas tend to have low coherence,
+because plants can grow or move from one acquisition to the next. Use caution when
+generating interferograms for areas with extensive/dense vegetation cover.
```

Comment:
I'd recommend re-ordering this content a bit to put the takeaway first:

```
Use caution when
generating interferograms for areas with extensive/dense vegetation cover. Because
Sentinel-1 is a C-band sensor, the waves will not penetrate very deeply into vegetation.
In densely vegetated areas, the imagery likely represents the top of the canopy rather
than the actual terrain. In addition, vegetated areas tend to have low coherence,
because plants can grow or move from one acquisition to the next. 
```